core/server/common/src/main/java/alluxio/master/journalv0/AsyncJournalWriter.java

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
optional: I would slightly prefer to use `alluxio.master.journal.v0` as the package